lib/lightning_web/live/run_live/workorder_component.ex

```diff
@@ -152,7 +153,14 @@ defmodule LightningWeb.RunLive.WorkOrderComponent do
               </h1>
               <span class="mt-2 text-gray-700">
                 <%= display_short_uuid(@work_order.id) %> .
-                <%= live_redirect to: Routes.project_dataclip_edit_path(@socket, :edit, @work_order.workflow.project_id, @work_order.reason.dataclip_id) do %>
+                <.link navigate={
+                  Routes.project_dataclip_edit_path(
```

Comment:
Same as my previous comment. Could we do `~p"/projects/#{@work_order.workflow.project_id}/dataclips/#{@work_order.reason.dataclip_id}/edit"` ?

Reply:
Done